README.md

```diff
@@ -1,59 +1,53 @@
-<p align="center">
-    <h1 align="center">Steampipe SQLite Extension</h1>
-</p>
+# Steampipe SQLite
 
-<p align="center">
-  <a aria-label="Steampipe logo" href="https://steampipe.io">
-    <img src="https://steampipe.io/images/steampipe_logo_wordmark_padding.svg" height="28">
-  </a>
-</p>
+A family of SQLite extensions, each derived from a [Steampipe plugin](https://hub.steampipe.io/plugins), that fetch data from cloud services and APIs.
 
-## Overview
+## Getting Started
 
-The `Steampipe SQLite Extension library` is a `SQLite3` extension that wraps [Steampipe](https://steampipe.io) plugins to interface with SQLite. 
+You can use an installer that enables you to choose a plugin and download the SQLite extension for that plugin. See the [installation docs](https://turbot.com/docs/steampipe_export/install) for details. 
```

Comment:
URL should be steampipe.io not turbot.com

Reply:
I also think it should link to the downloads page, which includes simple installation instructions.

Reply:
Suggest cutting the second sentence and just putting it on a paragraph by itself as:

**[Installation guide →](https://steampipe.io/TODO)**

Will give it more prominence, which I think it needs in the page.